### ISSUE TITLE
Unable to run testcafe from a npm script

### ISSUE THREAD

**Issue by warnellw** [DevExpress/testcafe#5398]
### What is your Test Scenario?
Following the installation guide and creating a simple test that is run via a `npm` script:
```
"scripts": {
  "test": "testcafe chrome **/*.js"
}
```
 
### What is the Current behavior?
- First time executing `npm run test`:
  - Unable to find module 'tape'.
- Second time executing `npm run test` after installing `tape`:
  - asar: 'chrome' is not an asar command. See 'asar --help'.
 
### What is the Expected behavior?
Running `testcafe` from a `npm` script should work exactly [as it's documented](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/install-testcafe.html) - the errors above should not be occurring. If `tape` is a peer-dependency of `testcafe`, it should be documented as part of the local installation process (i.e. `npm install testcafe tape --save-dev`).

Even after installing `tape`, `testcafe` should be able to run without hitting the `asar` error above. This is reproducible on both macOS and Windows. I am unable to get past the `asar` error.
 
### Error output

<details>
<summary>Full asar error output on macOS:</summary>

```
 asar: 'chrome' is not an asar command. See 'asar --help'.
rè
TAP version 13
# filter
ok 1 should be deeply equivalent
# skip over holes
ok 2 should be deeply equivalent
# modify
ok 3 should be deeply equivalent
# self
ok 4 should be strictly equal
# thisArg
ok 5 should be strictly equal
# thisArg native
ok 6 should be strictly equal
# typecheck
ok 7 arr undefined
ok 8 arr null
ok 9 fn wrong type
# can find an element
ok 10 should be strictly equal
# can find an element from an object
ok 11 should be deeply equivalent
# returns undefined if no callback returns true
ok 12 should be strictly equal
# setting object as thisArg
ok 13 this should be instanceof Car
ok 14 this should be instanceof Car
⠇
# loop should exit on first find
ok 16 element 0
ok 17 element 1
ok 18 element 2
# third callback argument array should be equal to first argument array
ok 19 should be deeply equivalent
ok 20 should be deeply equivalent
ok 21 should be deeply equivalent
# should throw TypeError if second argument is not a function
ERROR Cannot prepare tests due to an error.

ReferenceError: window is not defined
    at Object.<anonymous> (/Users/wes/Repositories/testapp/node_modules/atob/browser-atob.js:44:3)

Type "testcafe -h" for help.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! testapp@0.0.1 test: `testcafe chrome **/*.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the testapp@0.0.1 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/wes/.npm/_logs/2020-08-05T20_15_09_582Z-debug.log
```
</details>
 
### Your Environment details:
 
* testcafe version: `1.8.8`
* node.js version: `12.18.3`
* command-line arguments: `npm run test` (`testcafe chrome **/*.js`)
* browser name and version: `Chrome`
* platform and version: `macOS 10.15` `Windows 10 v2004`

**Comment by AlexSkorkin** [comment]
Hello,

Neither 'tape' nor 'asar' are direct TestCafe dependencies.

The issue occurs because you specified to run all tests from all folders including the 'node_modules' folder where third-party modules contain their own test cases not written using TestCafe.

For instance, the 'node_modules\array-find' module contains the 'test.js' file with the following `require` directive:

`var test = require('tape').test;`

That is why the 'tape' error occurs. The same scenario occurs with the 'asar' error for some other third-party module.

You might want to move your test cases to a separate folder to not run redundant tests from third-party modules and update your test script as exactly specified in the [documentation](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/install-testcafe.html):

```
"scripts": {
  "test": "testcafe chrome tests/"
}
```

**Comment by warnellw** [comment]
Thanks for the response! I've confirmed the glob pattern was the issue as you indicated. In this particular use case, testcafe tests are stored in a dedicated repository, so there is no top level `tests` folder (but creating one and moving the tests there is an option). I was able to use a glob pattern that excludes the `node_modules` folder (found [here](https://github.com/prettier/prettier/issues/1358#issuecomment-295669193)):

```
"scripts": {
  "test": "testcafe chrome '{,!(node_modules)/**/}*.js'"
}
```

Thanks again for your help!
